src/features/contracts/index.ts

```diff
@@ -198,13 +202,19 @@ export const synchronizeContracts = async (contractRepos: RepositoryInfo[]) => {
 	}
 
 	// We don't have automatic dependency resolution, so the order matters here.
-	for (const contractType of ['arch.sw', 'hw.device-type']) {
+	for (const contractType of [
+		'arch.sw',
+		'hw.device-manufacturer',
+		'hw.device-family',
+		'hw.device-type',
+	]) {
 		try {
 			const contracts = await getContracts(contractType);
 			await syncContractsToDb(contractType, contracts, globalSyncSettings);
 		} catch (err) {
 			console.error(
```

Comment:
Should this be a `captureException` that can also make its way to sentry?

Reply:
Yeah I think it makes sense here, since it's the top level and if this fails it means something went wrong. For the rest, I think having it in the logs is enough, otherwise it might spam Sentry.